astropy/cosmology/units.py

```diff
@@ -1,9 +1,19 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 """Cosmological units and equivalencies."""
 
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any, Literal
+
 import astropy.units as u
 from astropy.units.utils import generate_unit_summary as _generate_unit_summary
 
+if TYPE_CHECKING:
+    from collections.abc import Mapping
+
+    from astropy.cosmology.core import Cosmology
+    from astropy.units import Equivalency, Quantity
```

Comment:
The code is currently using `u.Equivalency`, so for consistency it would be better to import `Equivalency` normally and use that (instead of `u.Equivalency`) in the code, not just in the annotations.

Reply:
SGTM.